### DISCUSSION TITLE
For future reference: Lootables World Events level 70

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#5269] [Suggestions]
Mostly for the progression module, for future reference,
world events level 70 loot tables

brewfest: https://www.wowhead.com/wotlk/npc=23872/coren-direbrew
halloween: https://www.wowhead.com/wotlk/npc=23682/headless-horseman

summerfest: https://www.wowhead.com/wotlk/object=188124/ice-chest and https://www.wowhead.com/wotlk/object=188192/ice-chest

**Comment by Hihilist** [comment]
Getting them implemented as side versions of gear from badges, when we reach the Heroics phase would be very nice for gearing alts
